lib/buffer.js

```diff
@@ -237,6 +237,20 @@ Buffer.from = function from(value, encodingOrOffset, length) {
   );
 };
 
+// Identical to the built-in %TypedArray%.of(), but avoids using the deprecated
+// Buffer() constructor. Must use arrow function syntax to avoid automatically
+// adding a `prototype` property and making the function a constructor.
+//
+// Refs: https://tc39.github.io/ecma262/#sec-%typedarray%.of
+// Refs: https://esdiscuss.org/topic/isconstructor#content-11
+const of = (...items) => {
+  const newObj = createUnsafeBuffer(items.length);
+  for (var k = 0; k < items.length; k++)
+    newObj[k] = items[k];
+  return newObj;
+};
+Buffer.of = of;
```

Comment:
All the other functions in this file are defined as regular `function`s, not as arrow functions.
Is there a reason for specifically using an arrow function here? The non-arrow variant would have been one line shorter here.

Reply:
The reason is documented in the comment above :)

Reply:
Sorry, I missed that :-/.

Reply:
You can also use a normal function and throw an explicit error if `new.target` is truthy.

Reply:
@ljharb That doesn’t work with https://esdiscuss.org/topic/isconstructor#content-11 :)

Reply:
(We would want the function to *not* be a constructor.)

Reply:
Yes, but there's not really a difference to users between whether it's got `[[Construct]]` or whether it throws on `new`.